src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -1242,6 +1270,38 @@ bool CorProfiler::ProfilerAssemblyIsLoadedIntoAppDomain(AppDomainID app_domain_i
              managed_profiler_loaded_app_domains.end();
 }
 
+std::string CorProfiler::GetILCodes(std::string title, ILRewriter* rewriter,
```

Comment:
Given the issue we ran into, I think it'd make sense to also push the value of maxstack

Reply:
Yeah, I agree, I’ll add that

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/871/commits/2ea15cdc2f36f30a792c2ba14d22e6d7490100f6